lib/galaxy/util/dbkeys.py

```diff
@@ -38,7 +38,7 @@ def get_genome_build_names( self, trans=None ):
                 for dataset in datasets:
                     rval.append( ( dataset.dbkey, "%s (%s) [History]" % ( dataset.name, dataset.dbkey ) ) )
             user = trans.user
-            if user and 'dbkeys' in user.preferences:
+            if user and hasattr( user, 'preferences' ) and 'dbkeys' in user.preferences:
```

Comment:
I'm guessing this is because the toolshed reuses the user model (and this code), but doesn't have preferences defined or mapped?